bundles/org.openhab.binding.juicenet/README.md

```diff
@@ -0,0 +1,266 @@
+# JuiceNet Binding
+
+The JuiceNet binding will interface with the cloud portal to get status and manage your JuiceBox EV charger(s).
+In addition to getting the status of various items from the EV charger, it is also possible to start and stop charging sessions.
+
+## Supported Things
+
+This binding supports the following things:
+
+| thing               | type    | description                  |
+|----------           |-------- |------------------------------ | 
+| JuiceNet Account    | Bridge  | This represents the cloud account to interface with the JuiceNet API.  |
+| JuiceBox EV Charger | Device  | This interfaces to a specific JuiceBox EV charger associated with the JuiceNet account. |
+
+This binding should work with multiple JuiceBox EV chargers associated with the account, however it is currently only tested with a single EV charger.
+
+### Discovery
+
+Once a JuiceNet Account bridge has been created, any JuiceBox EV Chargers associated with this account will be discovered.
+
+
+### Thing Configuration
+
+The configuration required is to create a JuiceNet account thing and fill in the appropriate API token.
+The API token can be found on the Account page at https://home.juice.net/Manage.
+
+A JuiceBox EV Charger requires a a unitID which can also be found in the device settings at the JuiceNet web page.
+
+If configuring the binding with manual configuration an example thing file looks like this:
+
+```
+Bridge juicenet:account:myaccount [ apiToken="xxxx-xxxx-xxxx-xxxx-xxxxx" ] {
+    Thing device JamesCharger [ unitID="xxxxxxx" ]
+}
+```
```

Comment:
This should be moved below the **Channels** chapter and a header like "Example Things File" could be added. Or you could even have a small hierarchy:
- Level 2 header: Full Example
- Level 3 header: Things File
- Level 3 header: Items File

or similar.